src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+
+import { CdFormGroup } from '~/app/shared/forms/cd-form-group';
+import { ActionLabelsI18n } from '~/app/shared/constants/app.constants';
+
+import _ from 'lodash';
+
+@Component({
+  selector: 'cd-feedback',
+  templateUrl: './feedback.component.html',
+  styleUrls: ['./feedback.component.scss']
+})
+export class FeedbackComponent {
+  title = 'Feedback';
+  project: any = ['dashboard', 'block', 'objects', 'file_system', 'ceph_manager', 'orchestrator', 'ceph_volume', 'core_ceph'];
+  tracker: any = ['bug', 'feature'];
+  // severity: any = ['minor', 'major', 'critical']
+
+  feedbackForm: CdFormGroup;
+  constructor(
+    public activeModal: NgbActiveModal,
+    public actionLabels: ActionLabelsI18n
+  ) {
+    this.createForm();
+  }
+
+  private createForm() {
```

Comment:
If this form will stay a modal dialog in future, please consider using the [FormModalComponent](https://github.com/ceph/ceph/tree/master/src/pybind/mgr/dashboard/frontend/src/app/shared/components/form-modal). With this component you can define your form in a declarative style, no need to implement the form HTML manually. If the component does not provide all fields types you need in your form, please enhance it.